<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExpertDiscoveryPosNegMrkDialog</class>
 <widget class="QDialog" name="ExpertDiscoveryPosNegMrkDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>303</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Positive and Negative sequences markup</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/repeat_finder/repeats_image</normaloff>:/repeat_finder/repeats_image</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Positive sequences markup file</string>
       </property>
       <property name="buddy">
        <cstring>firstFileEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLineEdit" name="firstFileEdit"/>
     </item>
     <item row="3" column="3">
      <widget class="QToolButton" name="openFirstButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QCheckBox" name="oneSequenceCheckBox">
       <property name="text">
        <string>Generate description file</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QToolButton" name="openSecondButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="2">
      <widget class="QLineEdit" name="secondFileEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Negative sequences markup file</string>
       </property>
       <property name="buddy">
        <cstring>secondFileEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Description file</string>
       </property>
       <property name="buddy">
        <cstring>thirdFileEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="thirdFileEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="3">
      <widget class="QToolButton" name="openThirdButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QCheckBox" name="addToCurrentCheck">
       <property name="text">
        <string>Append to Current Markup</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QLabel" name="instructionLabel">
       <property name="text">
        <string>On this step you need to load markups for the sequences. 
Markaup is an annotation of a sequence with elementary signals.
Markup gives information where elementary signals are located in the sequences.

Complex signals will be build from the elementary signals and operations applied to them. 

Load markup for your sequences in specified XML format or genbank format.</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="4">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="4">
      <widget class="QCheckBox" name="lettersCheck">
       <property name="text">
        <string>Nucleotides Markup</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lettersCheck</tabstop>
  <tabstop>openFirstButton</tabstop>
  <tabstop>firstFileEdit</tabstop>
  <tabstop>openSecondButton</tabstop>
  <tabstop>secondFileEdit</tabstop>
  <tabstop>addToCurrentCheck</tabstop>
  <tabstop>openThirdButton</tabstop>
  <tabstop>thirdFileEdit</tabstop>
  <tabstop>oneSequenceCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExpertDiscoveryPosNegMrkDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExpertDiscoveryPosNegMrkDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
